output/clickhouse/clickhouse.go

```diff
@@ -0,0 +1,120 @@
+package outputclickhouse
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
```

Comment:
try `github.com/tsaikd/gogstash/config/goglog`

Reply:
I see not all plugins use it but if it's the desired one I will implement it.